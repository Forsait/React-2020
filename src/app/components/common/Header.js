// @flow
import React from 'react';
import type { Node } from 'react';
// $FlowFixMe
import { createUseStyles } from 'react-jss';

import Brand from './Brand';

const useStyles = createUseStyles({
  sub_header: {
    height: '64px',
    display: 'flex',
    justifyContent: 'space-between',
    alignItems: 'center',
    paddingLeft: '60px',
    paddingRight: '38px',
  },
});

/* eslint-disable */
type Props = {className?: string, children?: Node};

export default function Header(props: Props) {
  const classes = useStyles();
  const { className, children } = props;
  return (
    <div className={`${classes.sub_header} ${className || ''}`}>
      <Brand />
      {children}
    </div>
  );
}
