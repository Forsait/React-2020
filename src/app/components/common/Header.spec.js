import React from 'react';
import { shallow } from 'enzyme';

import Header from './Header';

describe('Header component', () => {
  it('should be render correctly', () => {
    const com = shallow(<Header />);
    expect(com).toMatchSnapshot();
  });
});
