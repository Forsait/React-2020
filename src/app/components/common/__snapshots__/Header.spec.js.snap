// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should be render correctly 1`] = `
<div
  className="undefined "
>
  <Brand />
</div>
`;
